ytfzf

```diff
@@ -464,10 +468,13 @@ start_ueberzug () {
     [ -e $FIFO ] || { mkfifo "$FIFO" || exit 1 ; }
     ueberzug layer --parser json --silent < "$FIFO" &
     exec 3>"$FIFO"
+    ueberzug_pid="$!"
 }
 stop_ueberzug () {
     exec 3>&-
     rm "$FIFO" > /dev/null 2>&1
+    # make sure ueberzug is really dead
+    /bin/kill $ueberzug_pid
```

Comment:
if there are 2 instances of ytfzf running, this will break thumbnails on all other instances